Source/inv.cpp

```diff
@@ -230,11 +232,8 @@ void DrawInv(const CelOutputBuffer &out)
 				CelBlitOutlineTo(out, GetOutlineColor(plr[myplr].InvBody[slot], true), RIGHT_PANEL_X + screen_x, screen_y, cel, celFrame, false);
 			}
 
-			if (plr[myplr].InvBody[slot]._iStatFlag) {
-				CelClippedDrawTo(out, RIGHT_PANEL_X + screen_x, screen_y, cel, celFrame);
-			} else {
-				CelDrawLightRedTo(out, RIGHT_PANEL_X + screen_x, screen_y, cel, celFrame, 1);
-			}
+			drawingFun = (plr[myplr].InvBody[slot]._iStatFlag ? &CelClippedDrawTo : &CelDrawLightRedTo);
+			drawingFun(out, RIGHT_PANEL_X + screen_x, screen_y, cel, celFrame);

```

Comment:
It looks like the function call is mostly controlled by the `_iStatFlag`.

Instead of assigning function pointers, let's add a wrapper:

```c++
void CelDrawWrapper(bool red, const CelOutputBuffer &out, int x, int y, const CelSprite &cel, int frame)
{
	if (red) {
		CelDrawLightRedTo(out, x, y, cel, celFrame);
	} else {
		CelClippedDrawTo(out, x, y, cel, celFrame);
	}
}
```

This is a bit more readable and easier to trace (reason about).

Just need a better name than `CelDrawWrapper`.

See comment in scrollrt.cpp for how it can be used there.